Jenkinsfile

```diff
@@ -33,7 +33,9 @@ pipeline {
             }
             post {
                 success {
-                    archiveArtifacts artifacts: '**/tds*.war', fingerprint: true, onlyIfSuccessful: true
+                    dir('artifacts/tds/build/libs/') {
```

Comment:
This shouldn't have 'artifacts' on the front, should just be 'tds/build/libs/'

Reply:
Fixed